### PR TITLE
move textures file prefix into TEXTURE dictionary

### PR DESCRIPTION
**Pull request by zhuyifengzju**

### CODE REVIEW

File: robosuite/utils/mjcf_utils.py [hunk 1]
```diff
@@ -95,34 +95,35 @@
...
-    "Bread": "bread.png",
-    "Can": "can.png",
-    "Ceramic": "ceramic.png",
-    "Cereal": "cereal.png",
-    "Clay": "clay.png",
-    "Dirt": "dirt.png",
-    "Glass": "glass.png",
-    "FeltGray": "gray-felt.png",
-    "Lemon": "lemon.png",
+    "WoodRed": DEFAULT_TEXTURE_PATH + "red-wood.png",
```
Comment (yukezhu) [inline robosuite/utils/mjcf_utils.py:100]: Please use `os.path.join` to concatenate file paths. simple string addition is error-prone, as it requires the "/" as the end of the folder path.

Comment (zhuyifengzju) [reply robosuite/utils/mjcf_utils.py]: I've changed the concatenating operation to using `os.path.join`.

File: robosuite/utils/mjcf_utils.py [hunk 2]
```diff
@@ -95,34 +95,35 @@
...
-    "Bread": "bread.png",
-    "Can": "can.png",
-    "Ceramic": "ceramic.png",
-    "Cereal": "cereal.png",
-    "Clay": "clay.png",
-    "Dirt": "dirt.png",
-    "Glass": "glass.png",
-    "FeltGray": "gray-felt.png",
-    "Lemon": "lemon.png",
+    "WoodRed": os.path.join(DEFAULT_TEXTURE_PATH, "red-wood.png"),
```
Comment (yukezhu) [inline robosuite/utils/mjcf_utils.py:100]: Lots of redundancy in the `TEXTURES` variable definition. please change it to Python list comprehension

Comment (zhuyifengzju) [reply robosuite/utils/mjcf_utils.py]: I've updated the modification to use list syntax.

**Comment by yukezhu** [comment]
This branch seems to be failing pytest. Please consult the [contributing guideline](https://github.com/ARISE-Initiative/robosuite/blob/master/CONTRIBUTING.md) and complete all the tests before submitting the PR.

**Review by yukezhu — APPROVED** [review]
LGTM

**Pull request merged**